packages/components/src/entities/SampleHeader.tsx

```diff
@@ -343,9 +348,8 @@ export const SampleHeaderImpl: FC<Props> = memo(props => {
                     onCancel={onHideModals}
                     maxSelected={1}
                     entityDataType={entityDataType}
-                    moveFn={api.entity.moveSamples}
-                    sourceContainer={sampleContainer}
```

Comment:
yeah "source" is confusing in the prop name now that it is something we are moving!